app/models/content/annotation.rb

```diff
@@ -94,7 +94,8 @@ def exists_in_published_casebook?
   def annotate_html inner, handle: true, final: false
     case kind
     when 'elide' then
-      "#{handle ? "<span class='annotate elide' data-annotation-id='#{id}'></span>" : ''}<span class='annotate elided' data-annotation-id='#{id}'>#{inner}</span>"
+      "#{handle ? "<button class='annotate elide' data-annotation-id='#{id}' aria-label='elided text' aria-expanded='false'></button>" : ''}" +
```

Comment:
Is `aria-label` where you were talking about all of your labels being the same?

Reply:
Yep, exactly. Without a `<label>` for the button, and without any text inside the button, this just gets read as... silence... by screen readers. `aria-label` is a way of providing a label for screen readers without changing the visual appearance of the page.